lesson/outline/outline-template.md

```diff
@@ -1,104 +1,323 @@
+    - test(‘description of desired behavior. ()=>{}, 3000)	
+
+
+### Checkpoints Summary
+
+
+1. Create an empty test function in the myproject.test.js file using the syntax covered above. 
+2. Describe the test 
+3. Declare the function using the default timeout 
+
+
+#### What is the purpose of these checkpoints?
+
+- The learner will be able to set up a test() function 
+
+### What would you like to have in the workspace for this exercise? Share your plan below.
+
+- I.E. text editor and output terminal
+
+## Exercise # 4: The `expect()` function
+
+### Which course outcomes will be covered by this exercise?
+
+1. Learners will be able write a basic unit test in jest using the expect() function in conjunction with five of types of matchers. 
+- Five types of matchers.
+    - .toBe and .toEqual
+    - .toContain
+    - .toBeTruthy
+    - .not
+    - .toBeDefined 
+
+
+### Narrative Summary
+
+1. In this lesson, we will cover the necessary basic syntax for writing our first complete unit test inside of our previously constructed test() function.
+2. Explain the simplicity of using jest syntax with an example of taking a sentence to describe the desired test and translating it into jest syntax.
+3. Demonstrate the different outcomes of the five selected matchers individually so, while encouraging the learner to visit documentation for the full list. 
 
 
 
 ### Checkpoints Summary
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/checkpoint/) for guidance on writing narratives for exercises._
 
-1. Checkpoint one
-2. Checkpoint two
+Using the previously created empty test function in the myproject.test.js file. 
+1. Write simple unit tests with .toBe
+2. Run test to verify
+3. Write simple unit tests with .not
+4. Run test to verify
+5. Write simple unit tests with .toBeTruthy
+6. Run test to verify
+7. Write simple unit tests with .toBeDefined
+8. Run test to verify
+7. Write simple unit tests with .toContain
+8. Run test to verify
+ 
+
 
 #### What is the purpose of these checkpoints?
 
+- The learner will write their first unit tests 
 
 
 ### What would you like to have in the workspace for this exercise? Share your plan below.
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/workspaces/) for guidance on writing narratives for exercises._
+- I.E. text editor and output terminal
 
-<hr>
 
-## Exercise # 3: _Insert exercise title here_
+## Exercise # 5: Testing Async Code in jest: Part 1
+
+### Which course outcomes will be covered by this exercise?
+
+1. Learners will be able to understand and solve false positives in jest using `done()`
+
+
+### Narrative Summary
+
+1. Begin by analyzing an api call that because of poor jest syntax, leads to a false positive, prematurely returning a test result.
+2. Explain why the test result returned as it did and how jest works with async code
+3. Re-write code to function properly
+Use done()
```

Comment:
Again, be more explicit here with the breakdown of the actual content structure of the exercise. Here's what I'm imagining.
1. Introduction of asynchronous code and the need to test it
2. Present an example of an asynchronous function call that we'd like to test.
3. Talk about how we might test it using what we know so far
4. Present an example of the naive test we might write and what the output of the `npm` test command would be 
5. Analyze it and explain how jest works with async code
6. Introduce the `done()` parameter 
7. Show a proper example of using `done()` to write the test
8. Explain the example

Reply:
Also, make sure to mention during this exercise that `done()` is useful when testing asynchronous functions that use callbacks to async responses.